### PR TITLE
Moved the updater into the button handler

### PR DESCRIPTION
**Pull request by radujipa**
Related to https://github.com/KanoComputing/peldins/issues/2067

### CODE REVIEW

**Comment by Ealdwulf** [comment]
:+1:

File: bin/kano-shutdown [hunk 1]
```diff
@@ -10,25 +10,26 @@
...
 from kano.gtk3.apply_styles import apply_styling_to_screen
-from kano.utils import run_cmd, run_bg
 
 from kano.paths import common_css_dir
+from kano.utils import run_cmd, run_bg
+
+# The steps needed before showing the dialog are a bit time consuming
+# so we raise the hourglass as soon as possible
+from kdesk.hourglass import hourglass_start, hourglass_end
+hourglass_start('kano-shutdown')
```
Comment (tombettany) [inline bin/kano-shutdown]: Was this not first so that the hourglass showed while the potentially lengthy imports were loaded?

Comment (radujipa) [inline bin/kano-shutdown]: I tested that and it doesn't make a difference, there are no lengthy imports here.

Comment (tombettany) [inline bin/kano-shutdown]: Fair enough

**Comment by tombettany** [comment]
:shipit:

**Comment by radujipa** [comment]
Thanks for the review guys :+1:

**Pull request merged**